crates/wasi-common/yanix/src/file.rs

```diff
@@ -1,25 +1,43 @@
 use crate::{cstr, from_result, from_success_code};
 use bitflags::bitflags;
 use cfg_if::cfg_if;
+#[cfg(not(any(target_os = "linux", target_os = "emscripten", target_os = "l4re")))]
+use libc::{
+    fstat as libc_fstat, fstatat as libc_fstatat, lseek as libc_lseek, openat as libc_openat,
+};
+#[cfg(any(target_os = "linux", target_os = "emscripten", target_os = "l4re"))]
```

Comment:
It's a microkernel with L4 ancestry. It's just here because I looked that Rust's `libc` crate to see which platforms needed this, and it was one of them.